futures/src/lib.rs

```diff
@@ -23,126 +25,25 @@
 
 #![no_std]
 
-#![deny(missing_docs, missing_debug_implementations, warnings)]
+#![deny(missing_docs, missing_debug_implementations)]
```

Comment:
Rustdoc for reexports seems broken. The crates all build fine individually (e.g. `cargo doc -p futures-core`). But in when reexported in the futures crate we get a bunch of warnings